modules/upgrade-cli.adoc

```diff
@@ -8,39 +8,42 @@
 = Upgrading your cluster through the CLI
 
 
-
 You can use the OpenShift Cluster Manager (OCM) CLI upgrade wizard to schedule automatic and manual upgrade policies, list the upgrade policy, and delete the upgrade policy for your {product-title} cluster.
 
+// TODO: Should there be a prereq here to have the ocm CLI installed? This is the first time I've come across it
```

Comment:
Honestly, engineering asked for this topic....but most customers use the UI for this or set up their upgrade policies at cluster creation. There is an OCM CLI guide being developed by the team handling the OCM docs.